democracy_club/templates/base.html

```diff
@@ -20,6 +20,8 @@
         {% block base_og_tags %}
             <meta property="fb:app_id" content="262795130596272" />
             <meta property="og:url" content="https://democracyclub.org.uk{{ request.path }}" />
+            <link rel="shortcut icon" href="{%  static "images/logo_icon.svg" %}">
```

Comment:
Not sure how much ordering is important but I'm feel like this is better placed with the other `<link rel` tags above than within this block